### PR TITLE
Default isolated install and FastDDS

### PR DESCRIPTION
**Pull request by pablogs9**
This PR changes the default `UAGENT_ISOLATED_INSTALL` to OFF to address https://github.com/eProsima/Micro-XRCE-DDS-Agent/issues/166 and changes the naming from FastRTPS to FastDDS.

Integration tests pass here: https://github.com/eProsima/Micro-XRCE-DDS/pull/51#issuecomment-636822136

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -75,9 +75,9 @@ if(UAGENT_P2P_PROFILE)
 endif()
 
 if(UAGENT_FAST_PROFILE)
-    set(_fastrtps_version 2.0.0)
-    set(_fastrtps_tag e58dcb1)
-    list(APPEND _deps "fastrtps\;${_fastrtps_version}")
+    set(_fastdds_version 2.0.0)
+    set(_fastdds_tag 2.0.x)
```
Comment (julibert) [inline CMakeLists.txt:79]: What happens if there is Fast-DDS change the patch version number in 2.0.x branch?

File: cmake/SuperBuild.cmake [hunk 2]
```diff
@@ -101,19 +101,19 @@ if(UAGENT_FAST_PROFILE)
             )
     endif()
 
-    # Fast RTPS.
-    unset(fastrtps_DIR CACHE)
-    find_package(fastrtps ${_fastrtps_version} EXACT QUIET)
-    if(NOT fastrtps_FOUND)
```
Comment (julibert) [inline cmake/SuperBuild.cmake:107]: I'm not sure it is going to work. Fast-DDS project name is still `fastrtps` so this find package has no effect.

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/222/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/221/)

File: cmake/SuperBuild.cmake (hunk 2 above)
Comment (pablogs9) [reply cmake/SuperBuild.cmake:107]: Fixed

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/223/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/222/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/224/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/223/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/225/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/224/)

**Comment by richiware** [comment]
Build status:
* Linux [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Linux/226/)
* Windows [Build Status](http://jenkins.eprosima.com:8080/job/Micro-XRCE-DDS-Agent%20Manual%20Windows/225/)

**Pull request merged**